pkg/nsx/services/mediator/mediator.go

```diff
@@ -67,8 +67,21 @@ func (serviceMediator *ServiceMediator) GetAvailableSubnet(subnetSet *v1alpha1.S
 			return *nsxSubnet.Path, nil
 		}
 	}
+	var tags []model.Tag
+	findLabelDefaultPodSubnetSet := false
+	for k, v := range subnetSet.Labels {
+		if k == common.LabelDefaultSubnetSet && v == common.LabelDefaultPodSubnetSet {
+			findLabelDefaultPodSubnetSet = true
+			break
+		}
+	}
+	if findLabelDefaultPodSubnetSet {
+		tags = append(tags, model.Tag{Scope: common.String(common.TagSubnetFor), Tag: common.String(common.LabelDefaultPodSubnetSet)})
```

Comment:
are these tags used for NetworkPolicy and SecurityPolicy? The tag values are "Pod" and "VirtualMachine"

Reply:
@dantingl means we define the value by ourselves.

Reply:
Yes, but please unify it with SecurityPolicy/NetworkPolicy usage.

Reply:
Here, only one tag: TagSubnetFor, it might be sufficient for securtiypolicy, but not good enough for network policy, Since networkpolciy need to use namespace_uid of podsubnet, and namespace_name of podsubnet to select pod segment. that is to say, using ncp/project_uid (namespace uid of segment) to select pod segment, using ncp/project(namesapce name) as well. 
In our case, we need to use nsx-op/createfor: pod to check if it's a pod subnet and then plus one more condition to nsx-op/namespace to select the corresponding subnet allocated to this namespace. 

Hence, I suggest we follow ncp's implementation, 
we can add two specific tags: nsx-op/pod_subnet_namespace_uid and nsx-op/pod_subnet_namespace to the pod subnet, this way will facilitate selection of  pod subnet, since we know these two tags are only for pod subnet. 

As for vmSubnet,  we can use nsx-op/namespace_uid and nsx-op/namespace tags, we can assume these two common tags are specific for vmSubnet. So, we don't need to add ncp/vif_project_uid and ncp/vif_project as we did in NCP to distinguish vm segment. 
whereas, we need to remove nsx-op/namespace from basic tag in the subnet buildBasicTags
https://github.com/vmware-tanzu/nsx-operator/blob/vpc_dev/pkg/nsx/services/subnet/builder.go#L79

Reply:
fixed.